code/game/gamemodes/dynamic/dynamic_storytellers.dm

```diff
@@ -0,0 +1,127 @@
+				continue
+			rule.trim_candidates()
+			if (rule.ready())
+				var/property_weight = 0
+				for(var/property in property_weights)
+					if(property in rule.property_weights)
+						property_weight += rule.property_weights[property] * property_weights[property]
+				drafted_rules[rule] = rule.get_weight() + property_weight
+	return drafted_rules
+
+/datum/dynamic_storyteller/proc/latejoin_draft(mob/living/carbon/human/newPlayer)
+	var/list/drafted_rules = list()
+	for (var/datum/dynamic_ruleset/latejoin/rule in mode.latejoin_rules)
+		if (rule.acceptable(mode.current_players[CURRENT_LIVING_PLAYERS].len, mode.threat_level) && mode.threat >= rule.cost)
+			// Classic secret : only autotraitor/minor roles
+			if (GLOB.dynamic_classic_secret && !((rule.flags & TRAITOR_RULESET) || (rule.flags & MINOR_RULESET)))
+				continue
+			// No stacking : only one round-ender, unless threat level > stacking_limit.
+			if (mode.threat_level > GLOB.dynamic_stacking_limit && GLOB.dynamic_no_stacking)
+				if(rule.flags & HIGHLANDER_RULESET && mode.highlander_executed)
+					continue
+
+			rule.candidates = list(newPlayer)
+			rule.trim_candidates()
+			if (rule.ready())
+				var/property_weight = 0
+				for(var/property in property_weights)
+					if(property in rule.property_weights)
+						property_weight += rule.property_weights[property] * property_weights[property]
+				drafted_rules[rule] = rule.get_weight() + property_weight
+	return drafted_rules
+
+/datum/dynamic_storyteller/proc/event_draft()
+	var/list/drafted_rules = list()
+	for(var/datum/dynamic_ruleset/event/rule in mode.events)
+		if(rule.acceptable(mode.current_players[CURRENT_LIVING_PLAYERS].len, mode.threat_level) && mode.threat >= rule.cost)
+			if(rule.ready())
+				var/property_weight = 0
+				for(var/property in property_weights)
+					if(property in rule.property_weights)
+						property_weight += rule.property_weights[property] * property_weights[property]
+				drafted_rules[rule] = rule.get_weight() + property_weight
+	return drafted_rules
+
+/datum/dynamic_storyteller/team
+	name = "Teamwork"
+	desc = "Rules that are likely to get the crew to work together against a common enemy."
+	curve_centre = 2
+	curve_width = 1.5
+	property_weights = list("valid" = 3, "trust" = 5)
+
+/datum/dynamic_storyteller/liteextended
+	name = "Extended-lite"
+	desc = "Rules that are likely to lead to rounds that reach their finish at the shuttle autocall."
+	curve_centre = -5
+	curve_width = 0.5
+	property_weights = list("extended" = 5, "chaos" = -1, "valid" = -1, "story_potential" = 1, "conversion" = -10)
+
+/datum/dynamic_storyteller/memes
+	name = "Story"
+	desc = "Rules that might lead to fun stories to tell."
+	curve_width = 4
+	property_weights = list("story_potential" = 10)
+
+/datum/dynamic_storyteller/cowabunga
+	name = "Cowabunga"
+	curve_centre = 10
+	desc = "The most chaotic rules only. A validhunter's dream."
```

Comment:
Saying "A validhunter's dream." is pretty much a sucker to those who complain about antagonists being hunted too much. And considering this also drafts in some potentially dangerous random events. I guess the point is more of a wild ride than a witch hunt.
```suggestion
	desc = "The most chaotic rules only. Expect the unexpectable."
```
P.S. The suggested description is just an example, and a reference to some ugandan meme. 

I'll perhaps review the rest later.

Reply:
Just change it to "Cowabunga it is"